pandas/core/tools/datetimes.py

```diff
@@ -726,13 +729,30 @@ def to_datetime(
             with year first.
 
     utc : bool, default None
-        Return UTC DatetimeIndex if True (converting any tz-aware
-        datetime.datetime objects as well).
+        Control timezone-related parsing, localization and conversion.
+
+        - If ``True``, the function *always* returns a timezone-aware
+          UTC-localized Timestamp, Series or DatetimeIndex. To do this,
+          timezone-naive inputs are *localized* as UTC (e.g.
+          ``01-01-2020 01:00:00`` becomes ``01-01-2020 01:00:00Z``), while
+          timezone-aware inputs are *converted* to UTC (e.g.
+          ``01-01-2020 01:00:00+0100`` becomes ``01-01-2020 00:00:00Z``).
+
+        - If ``False`` (default), the result is a "best effort automation",
```

Comment:
This description doesn't seem to describe much IMO.

How about `inputs will not be attempted to be explicitly coerced to UTC`

Reply:
Indeed this was a bit clunky. Leveraging your proposal, I wrote `inputs will not be coerced to UTC` but changed a bit the details.